### PR TITLE
lint fix

### PR DESCRIPTION
**Pull request by harishvanga333**
Resolves #{github issue number}

### Description
Please explain the changes you made here.

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan
1. Go to ...

- [ ] For higher-risk changes: [Deploy the custom branch to UAT to test](https://github.com/department-of-veterans-affairs/appeals-deployment/wiki/Applications---Deploy-Custom-Branch-to-UAT)

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---

### Code Documentation Updates
- [ ] Add or update code comments at the top of the class, module, and/or component.

### Storybook Story
*For Frontend (Presentationa) Components*
* [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file (e.g. create `MyComponent.stories.js` alongside `MyComponent.jsx`)
* [ ] Give it a title that reflects the component's location within the overall Caseflow hierarchy
* [ ] Write a separate story (within the same file) for each discrete variation of the component

### Database Changes
*Only for Schema Changes*

* [ ] Add typical timestamps (`created_at`, `updated_at`) for new tables
* [ ] Update column comments; include a "PII" prefix to indicate definite or potential [PII data content](https://github.com/department-of-veterans-affairs/appeals-team/blob/master/caseflow-team/0-how-we-work/pii-handbook.md#what-is-pii)
* [ ] Have your migration classes inherit from `Caseflow::Migration`, especially when adding indexes (use `add_safe_index`) (see [Writing DB migrations](https://github.com/department-of-veterans-affairs/caseflow/wiki/Writing-DB-migrations))
* [ ] Verify that `migrate:rollback` works as desired ([`change` supported functions](https://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method))
* [ ] Perform query profiling (eyeball Rails log, check bullet and fasterer output)
* [ ] Add appropriate indexes (especially for foreign keys, polymorphic columns, unique constraints, and Rails scopes)
* [ ] Run `make check-fks`; add any missing foreign keys or add to `config/initializers/immigrant.rb` (see [Record associations and Foreign Keys](https://github.com/department-of-veterans-affairs/caseflow/wiki/Record-associations-and-Foreign-Keys))
* [ ] Add `belongs_to` for associations to enable the [schema diagrams](https://department-of-veterans-affairs.github.io/caseflow/task_trees/schema/schema_diagrams) to be automatically updated
* [ ] Post this PR in #appeals-schema with a summary
* [ ] Document any non-obvious semantics or logic useful for interpreting database data at [Caseflow Data Model and Dictionary](https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Data-Model-and-Dictionary)

### Integrations: Adding endpoints for external APIs
* [ ] Check that Caseflow's external API code for the endpoint matches the code in the relevant integration repo
  * [ ] Request: Service name, method name, input field names
  * [ ] Response: Check expected data structure
* [ ] Update Fakes
* [ ] Integrations impacting functionality are tested in Caseflow UAT

### CODE REVIEW

File: lib/helpers/cancel_active_task_array.rb [hunk 1]
```diff
@@ -1,69 +1,70 @@
...
+      user = assigned_to_id.to_i
+      puts("Checking for a user or organization...")
+      # Show the user if found.
+      if User.find_by_id(user).nil?
+        puts("#{user} User Not found")
+        return
+      end
+
+      # Show the organization if found.
+      if !Organization.find_by_id(user).nil?
```
[the remark this answers is not in the document]
Comment (AimanK) [reply lib/helpers/cancel_active_task_array.rb:21]: @harishvanga333

**Review by AimanK — COMMENTED** [review]
Made a comment

File: lib/helpers/cancel_active_task_array.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (harishvanga333) [reply lib/helpers/cancel_active_task_array.rb:21]: @AimanK if it didn't find the the organization we need to print puts statement and we need to return?

[the remark this answers is not in the document]
Comment (AimanK) [reply lib/helpers/cancel_active_task_array.rb:21]: In the proposed changes, we are having the puts print when an organization is found when it should be only shown when an organization is not found. if the organization is found, nothing happens (empty return). Then the else with the puts inside of it should only be printed if an organization is not found.

[the remark this answers is not in the document]
Comment (AimanK) [reply lib/helpers/cancel_active_task_array.rb:21]: I understand the reason for the confusion, the comment on line 20 does not represent the code below correctly, I am not sure who wrote/approved it

File: lib/helpers/cancel_active_task_array.rb [hunk 2]
```diff
@@ -1,69 +1,70 @@
...
+        puts("Unable to find organization #{user} based upon Organization.find_by_id.")
+        return
+      end
+
+      # Checks if the assigned_to_id is found in the User or organization table
+      if !User.find_by_id(user).nil? || !Organization.find_by_id(user).nil?
+        puts("Unable to find user or organization by the assigned_to_id of #{user}")
```
Comment (AimanK) [inline lib/helpers/cancel_active_task_array.rb:28]: This is similar to the last issue above as well, this puts needs to be placed under an `else`

Comment (AimanK) [reply lib/helpers/cancel_active_task_array.rb:28]:
```
else
            puts ("Unable to find user or organization by the assigned_to_id of #{user}");
                fail interupt;
```

Like so above ^. I also believe that the `fail interupt` is a typo from the previous contributor and needs to be fixed to `fail Interrupt` as well as it is important to trigger a run-time error if we are unable to find a user or an organization

**Review by AimanK — COMMENTED** [review]
I have left another comment, everything else in this file is good to be merged after these small changes are made

**Review by AimanK — APPROVED** [review]
This is good to be merged

**Pull request merged**